content/searching/index.md

```diff
@@ -783,3 +783,107 @@ Now let's add some commenting
 beginning of the regex")(?R) \) )* [^()]* )
 
 ~~~
+
+## Searching actions when recorded as macros
+
+The Find family of actions can be recorded in a macro to make them easy to name and later replay via the **Macro** menu or an assigned keycombination.  Somewhat unfortunately, **Find what** and **Replace with** text is hardwired into the macro when it is created, and isn't something the user can change when the macro runs, but often this isn't a significant limitation.
+
+Note, however, that Find-related actions are recorded a bit differently than other Notepad++ actions, so we'll discuss them a bit more in-depth here.  Typically, Notepad++ will record a step in a macro every time a user does something in the Notepad++ user interface.  The Find family of actions is more "coordinated" where macro recording is concerned.
+
+The macro recorder only records when an actual Find family action (e.g. **Replace**, **Find All in Current Document**, etc.) occurs.  Thus you can tweak a future action's parameters (e.g. **Match case**, **Wrap around**, etc.) all you'd like, and all of that fiddling doesn't get remembered.  At the point where you peform an action, then a snapshot is taken of all of the parameters and the action, and this is logged in the macro memory as a proper macro sequence.
+
+While the user can simply record and use Find family macros, one can also edit those macros later to change or add to their functionality, so it is helpful to know the details of the macro sequences that were previously recorded.  While the details of how macros in general are recorded and stored in *shortcuts.xml* is discussed elsewhere, here are the details of what happens when Notepad++ saves a recorded Find family macro:
```

Comment:
I'd suggest changing `discussed elsewhere` to `[discussed elsewhere](../macros/)`: I've been using cross-document links when I do my edits, and I think it makes sense to have one here, too.

Actually, _is_ it really discussed on the Macros page?  If I'm reading correctly, not yet.  Probably need to add it to our TODO list to bring in the old macros-nitty-gritty from the Wiki to the Macro page.

Reply:
Okay, I'll update it with the cross-link when I next hit the document (I have more to do), since you are so "easy" on me.  

I didn't do it because the macros document is basically empty, so not much to jump to.  In this early stage of documentation revision it is a bit hard to do good cross linking; things are evolving.

Yes, I *presumed* that it will be discussed on the upcoming Macros pages.